service/src/main/java/io/camunda/service/ClusterVariableServices.java

```diff
@@ -0,0 +1,120 @@
+import java.util.concurrent.CompletableFuture;
+import org.agrona.concurrent.UnsafeBuffer;
+
+public final class ClusterVariableServices
+    extends SearchQueryService<
+        ClusterVariableServices, ClusterVariableQuery, ClusterVariableEntity> {
+
+  private final ClusterVariableSearchClient clusterVariableSearchClient;
+
+  public ClusterVariableServices(
+      final BrokerClient brokerClient,
+      final SecurityContextProvider securityContextProvider,
+      final ClusterVariableSearchClient clusterVariableSearchClient,
+      final CamundaAuthentication authentication,
+      final ApiServicesExecutorProvider executorProvider,
+      final BrokerRequestAuthorizationConverter brokerRequestAuthorizationConverter) {
+    super(
+        brokerClient,
+        securityContextProvider,
+        authentication,
+        executorProvider,
+        brokerRequestAuthorizationConverter);
+    this.clusterVariableSearchClient = clusterVariableSearchClient;
+  }
+
+  @Override
+  public ClusterVariableServices withAuthentication(final CamundaAuthentication authentication) {
+    return new ClusterVariableServices(
+        brokerClient,
+        securityContextProvider,
+        clusterVariableSearchClient,
+        authentication,
+        executorProvider,
+        brokerRequestAuthorizationConverter);
+  }
+
+  public CompletableFuture<ClusterVariableRecord> createGloballyScopedClusterVariable(
+      final String name, final Object value) {
+    return sendBrokerRequest(
+        new BrokerCreateClusterVariableRequest()
+            .setName(name)
+            .setValue(new UnsafeBuffer(MsgPackConverter.convertToMsgPack(value)))
+            .setGlobalScope());
+  }
+
+  public CompletableFuture<ClusterVariableRecord> createTenantScopedClusterVariable(
+      final String name, final Object value, final String tenantId) {
+    return sendBrokerRequest(
+        new BrokerCreateClusterVariableRequest()
+            .setName(name)
+            .setValue(new UnsafeBuffer(MsgPackConverter.convertToMsgPack(value)))
+            .setTenantScope(tenantId));
+  }
+
+  public CompletableFuture<ClusterVariableRecord> deleteGloballyScopedClusterVariable(
+      final String name) {
+    return sendBrokerRequest(
+        new BrokerDeleteClusterVariableRequest().setName(name).setGlobalScope());
+  }
+
+  public CompletableFuture<ClusterVariableRecord> deleteTenantScopedClusterVariable(
+      final String name, final String tenantId) {
+    return sendBrokerRequest(
+        new BrokerDeleteClusterVariableRequest().setName(name).setTenantScope(tenantId));
+  }
+
+  public ClusterVariableEntity getGloballyScopedClusterVariable(final String name) {
+    return executeSearchRequest(
+        () ->
+            clusterVariableSearchClient
+                .withSecurityContext(
+                    securityContextProvider.provideSecurityContext(
+                        CamundaAuthentication.anonymous()))
```

Comment:
❓ Users don't need any permissions to read the variables? Or will this follow later?